package.json

```diff
@@ -119,16 +119,14 @@
 	"scripts": {
 		"vscode:prepublish": "npm run compile",
 		"compile": "tsc -p ./",
-		"watch": "tsc -watch -p ./",
-		"postinstall": "node ./node_modules/vscode/bin/install",
-		"test": "npm run compile && node ./node_modules/vscode/bin/test"
```

Comment:
This doesn't work with the new `@types/vscode` dependency. I found no tests in this project, so I'm removing this for now. This needs to be properly updated if/when tests are added: https://code.visualstudio.com/api/working-with-extensions/testing-extension#migrating-from-vscode